From: Tristan Seligmann <mithrandi@debian.org>
Date: Mon, 3 Aug 2020 15:12:37 +0200
Subject: Disable hoverxref; not in Debian.

---
 hypothesis-python/docs/conf.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hypothesis-python/docs/conf.py b/hypothesis-python/docs/conf.py
index 9ce1081..5dc9252 100644
--- a/hypothesis-python/docs/conf.py
+++ b/hypothesis-python/docs/conf.py
@@ -30,8 +30,6 @@ extensions = [
     "sphinx.ext.extlinks",
     "sphinx.ext.viewcode",
     "sphinx.ext.intersphinx",
-    "hoverxref.extension",
-    "sphinx_selective_exclude.eager_only",
 ]
 
 templates_path = ["_templates"]
